pom.xml

```
@@ -57,7 +56,7 @@
     <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
     <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
     <url>https://github.com/${gitHubRepo}</url>
-    <tag>${scmTag}</tag>
+    <tag>gitlab-plugin-1.7.1</tag>
```

This is used for the incremental build process.  We don't want to make this change.
```suggestion
    <tag>${scmTag}</tag>
```